style2.css

```diff
@@ -1,261 +1,261 @@
-    left: 12px;
-    top: 11px;
-    background: #ff6b00;
-    margin-block-end: auto;
-    margin-right: 12px;
-    margin-top: 12px;
-  }
-
-  .circle {
-    width: 18px;
-    height: 18px;
-    left: 12px;
-    top: 11px;
-    background: #ff6b00;
-    border-radius: 150px;
-    margin-block-end: auto;
-    margin-right: 12px;
-  }
-
-  .about-discription-container {
-    display: flex;
-    margin-top: 146px;
-  }
-
-  .shapes-daimond {
-    margin-top: -90px;
-    margin-left: 20px;
-  }
-
-  .shapes-rectangle {
-    margin-left: 120px;
-  }
-
-  .shapes-circle {
-    margin-left: 120px;
-  }
-
-  .about-container h3 {
-    font-family: Inter, sans-serif;
-    font-style: normal;
-    font-weight: 400;
-    font-size: 22px;
-    line-height: 43px;
-    display: flex;
-    align-items: center;
-    text-align: center;
-    color: white;
-    margin-left: 120px;
-    justify-content: space-between;
-    margin-bottom: auto;
-  }
-
-  .circle-container {
-    margin-left: 120px;
-    display: block ruby;
-  }
-
-  .rectangle-container {
-    margin-left: 115px;
-    display: block ruby;
-  }
-
-  /* stylelint-disable-next-line no-duplicate-selectors */
-}
+/* stylelint-disable selector-type-no-unknown */
+@media (min-width: 768px) {
+  html,
+  body {
+    width: 100vw;
+  }
+
+  #drop {
+    display: none;
+  }
+
+  .name {
+    margin-left: 154px;
+  }
+
+  .desktop-version {
+    display: ruby;
+    padding-right: 156px;
+    color: white;
+  }
+
+  .desktop-version img {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-top: 55px;
+  }
+
+  .desktop-menu {
+    margin-right: 32px;
+  }
+
+  .upper-body-text h1 {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    letter-spacing: 0.37px;
+    color: #ff6b00;
+    height: 123px;
+  }
+
+  .upper-body-text h2 {
+    color: #fff5e1;
+    font-family: Roboto, sans-serif;
+    font-size: 32px;
+    line-height: 44px;
+    height: 27px;
+  }
+
+  .upper-body-text p {
+    color: #fff;
+    font-family: Poppins, sans-serif;
+    font-size: 20px;
+    line-height: 32px;
+    height: 123px;
+    margin-right: 732px;
+  }
+
+  /* stylelint-disable-next-line no-duplicate-selectors */
+  .upper-body-text h1 {
+    margin-right: 732px;
+  }
+
+  .upper-body-text {
+    margin-left: 153px;
+  }
+
+  .upper-body {
+    background-image: url("upperimage.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+  }
+
+  .myrecent {
+    display: flex;
+    margin-left: 153.5px;
+  }
+
+  .myrecent hr {
+    margin-top: 161px;
+    width: 794px;
+    border: 1px solid #212121;
+  }
+
+  .middle-image1 {
+    display: flex;
+  }
+
+  .myrecent h1 {
+    margin-top: 135px;
+    height: fit-content;
+    margin-left: 15px;
+  }
+
+  .middle-image {
+    display: none;
+  }
+
+  .middle-section {

```

Comment:
- [ ] Please fix the `see project` button positioning and try to stick as much as possible to the given Figma design.
and fix it in the mobile version as well.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/160947954-3101e97a-f7c0-4252-ab1c-438af25b5b8c.png)  | ![image](https://user-images.githubusercontent.com/68796232/160947969-f27d2f36-d4be-4814-a5a4-0dd37d4a3a48.png) |